libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Preds/PParams.hs

```diff
@@ -51,6 +52,16 @@ nonNegativeInterval r = case (boundRational @NonNegativeInterval r) of
   Just nn -> nn
   Nothing -> error ("Can't make NonNegativeInterval from: " ++ show r)
 
+{- Preds we want to be True in pparams
+  [ Sized (AtLeast 1) (maxBHSize proof)
+  , Sized (AtLeast 1) (maxTxSize proof)
+  , SumsTo (Right 1) (maxBBSize proof) LTE [One (maxBHSize proof), One (maxTxSize proof)]
+  , Component
+      (Right (pparams proof))
+      [field pp (maxTxSize proof), field pp (maxBHSize proof), field pp (maxBBSize proof)]
+  ]
+-}
```

Comment:
Dead code?
```suggestion
```

Reply:
This comment is meant as a statement of intent
 The actual code achieves this algorithmically, but we want this declaration of what should be true